src/View/mainpage/StructureWidget.py

```diff
@@ -55,7 +54,8 @@ def checkbox_clicked(self, checked):
 
     def roi_suggestions(self):
         """
-        Get the top 3 suggestions for the selected ROI based on string matching with standard ROIs provided in .csv format.
+        Get the top 3 suggestions for the selected ROI based on string matching with standard ROIs provided in .csv
```

Comment:
To comply with PEP 8, comments and docstrings can only have a maximum of 72 characters per line.

Reply:
Please address this.

Reply:
done